src/components/Tile/tile.scss

```diff
@@ -0,0 +1,176 @@
+//===================
+//SMALL TILE STYLES
+//===================
+.tile--sm {
+  display: flex;
+  flex-direction: column;
+
+  & > .tile--text-container {
+    padding-left: 0;
+
+    & > p {
+      font-size: calc(14px + 2 * ((100vw - 320px) / 1264));
+    }
+  }
+
+  @include max-breakpoint('lg'){
+    margin-bottom: rem(20px);
+  }
+}
+
+.title--sm {
+  margin-bottom: rem(14px);
+  @include type-style('productive-heading-04');
+  font-size: calc(20px + 8 * ((100vw - 320px) / 1264));
+}
+
+.img--sm {
+  width: 100%;
+  height: auto;
+  margin-bottom: rem(18px);
+
+  & > .gatsby-resp-image-wrapper {
+    max-width: rem(500px) !important;
```

Comment:
when I removed the `max-width` for `tile--sm` and `img--sm` the image was still only `288px`, realized the gatsby wrapper `max-width` was following the img file size (288) so I set a way larger `max-width` so it'll take up the full width of the column.

Reply:
I had to add the same thing to my PR. added in the pages.scss file

```
.gatsby-resp-image-wrapper {
  max-width: 100% !important; //need to see why the plugin isn't respectinng the max width. this fixes for now. 🤔
}
```